src/server/models/listItem/summary.helpers.ts

```diff
@@ -130,3 +131,12 @@ export function getActivityStatus(item: ListItemWithHistory): ActivityStatusView
 
   return statusToActivityVM[status];
 }
+
+export function getLastPublished(events: Array<{ type: string; time: Date }> | undefined): string {
```

Comment:
this should just be part of the query, or a query on events. but I'm pretty sure it should be ordered in desc anyway if using findByListItemId. 


```
history: {
          orderBy: {
            time: "desc",
          },
        },
```

or 

```
function getDateOfMostRecentEvent(type: ListItemEvent, id: ListItem["id"]) {
  return prisma.event.findFirst({
    where: {
      listItemId: id,
      type,
    },
    orderBy: {
      time: "desc"
    },
  })
}

```